src/components/common/PlantingSiteMapLegend.tsx

```diff
@@ -89,6 +89,8 @@ export default function PlantingSiteMapLegend({ options }: PlantingSiteMapLegend
     theme.palette.TwClrBaseBlue300,
     theme.palette.TwClrBasePink300,
     theme.palette.TwClrBaseYellow300,
+    theme.palette.TwClrBaseOrange300,
+    hasAdHocPlot,
```

Comment:
NIT You can replace all the `theme.palette.` dependencies in this `useMemo` with just `theme`. Just a preference though